package/MDAnalysis/core/topologyattrs.py

```diff
@@ -2343,7 +2343,6 @@ class _ConnectionTopologyAttrMeta(_TopologyAttrMeta):
     to return only the connections within the atoms in the group.
     """
     def __init__(cls, name, bases, classdict):
-        type.__init__(type, name, bases, classdict)
```

Comment:
I guess the call just gets inherited from the parent class anyways (given we are calling super below)?

@richardjgowers - looks like you're the original author of the parent class, does it all make sense to you?

Reply:
This seems to work